package.json

```diff
@@ -51,9 +51,10 @@
   "peerDependencies": {
     "react": "^16.6.1 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.6.1 || ^17.0.0 || ^18.0.0",
-    "styled-components": "^5.1.0"
+    "styled-components": "^5.1.0 || ^6.0.0"
   },
   "dependencies": {
+    "@emotion/is-prop-valid": "^1.2.1",
```

Comment:
Is there some rationale for this being a direct dependency rather than a peer dependency?  It is unintuitive (to me, at least) that a package used specifically for a peer dependency would be a direct dependency.

Reply:
@jcfilben Do we need to add this directly? Or is it included with styled-components v6?

Reply:
It's not included, but it can be configured globally through styled components (and likely will in many cases). The alternative is to migrate to transient props, which a lot of people seem to be balking at

Reply:
@emotion/is-prop-valid is a dependency of styled-components but since we need to import and use `isPropValid` within our own code I added it as a dependency on Grommet. Although it is a styled-components dependency, I didn't see a way to import `isPropValid` through styled-components

Reply:
v6 dropped the direct dependency, making it a peer dependency. They migrated to transient props instead

https://styled-components.com/docs/faqs#shouldforwardprop-is-no-longer-provided-by-default

Reply:
We explored transient props but ultimately decided to use is-prop-valid to filter out styling props from being passed to the dom. Some more context on our explorations:
https://github.com/grommet/grommet/pull/6947
https://github.com/grommet/grommet/pull/6941
https://github.com/grommet/grommet/tree/styled-components-v6 - branch where we started converting to transient props

Reply:
Thanks for jogging my memory Jessica!